### PR TITLE
updating Body, Part and Assembly icons

### PR DESCRIPTION
**Pull request by pgilfernandez**
Updating **Body** and **Part** new icons and suggesting a new **Assembly** icon as discussed in FreeCAD forum: https://forum.freecadweb.org/viewtopic.php?f=34&t=23625&sid=33360e767363f724cc1584a764e95d61

Also, merge it always after PR 916 (https://github.com/FreeCAD/FreeCAD/pull/916) because it has slight icon design updates.

Thanks

### CODE REVIEW

File: src/Gui/Stylesheets/Dark-green.qss [hunk 1]
```diff
@@ -1680,8 +1692,8 @@ QListWidget::indicator:checked:selected,
 QListWidget::indicator:indeterminate:selected,
 QTreeView::indicator:checked:selected,
 QTreeView::indicator:indeterminate:selected {
-    background-color: #a5c61a; /* slightly lighter than default */
-    border-color: #74831d; /* slightly darker than default */
+    background-color: #a5c61a; /* slighly lighter than default */
+    border-color: #74831d; /* slighly darker than default */
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Dark-green.qss]: Slightly is correct. Not slighly

File: src/Gui/Stylesheets/Dark-orange.qss [hunk 2]
```diff
@@ -1666,7 +1678,7 @@ QTreeView::indicator {
     border-radius:2px;
 }
 
-/* fix for QTreeView::indicator losing its margin */
+/* fix for QTreeView::indicator loosing its margin */
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Dark-orange.qss]: Losing not loosing

**Comment by pgilfernandez** [comment]
Uff, I had troubles setting up the rebase with the computer I have here (I'm in vacations) so I probably did it uncorrectly... lol
I'll try to solve it tomorrow paying more attention

**Comment by pgilfernandez** [comment]
Please, forget this PR, I messed up the branch and it's no more available. I have created a new one where even the stylesheets typos are fixed (thanks @luzpaz):
https://github.com/FreeCAD/FreeCAD/pull/965

I think it's finally everything is fine...

**Comment by wwmayer** [comment]
Replaced by PR 965

**Pull request closed without merging**